src/elements/alert/alert/alert.ts

```diff
@@ -41,6 +41,8 @@ export class SbbAlertElement extends SbbIconNameMixin(LitElement) {
   public static readonly events = {
     willOpen: 'willOpen',
     didOpen: 'didOpen',
+    willClose: 'willClose',
```

Comment:
Since you added these two events, maybe you could extend the `SbbOpenCloseBaseElement`